en/_posts/2015-12-29-mroonga-5.11.md

```diff
@@ -28,7 +28,7 @@ Supported MySQL 5.6.28 and 5.7.10 for RPM packages (mysql56-community-mroonga an
 
 #### Supported index table in `match_columns` in script syntax
 
-An index table name uses `-` (hyphen) as separator untile Mroonga 5.10. But `-` can't be used in `match_columns` in script syntax (`mroonga_command`), so index table can't be used in `match_columns` in script syntax. Since Mroonga 5.11, index table name separator is changed to `#` from `-`. It makes that index table can be used in `match_columns` in script syntax. However, there is no impact for many users who uses Mroonga via SQL normally.
+Before Mroonga 5.10, `-` (hyphen) was used as separator character against an index table. But `-` could't be used in `match_columns` in script syntax (`mroonga_command`), so index table could't be used in `match_columns` in script syntax. Since Mroonga 5.11, index table name separator has been changed to `#` from `-`. It makes that index table can be used in `match_columns` in script syntax. However, there is no impact for many users who uses Mroonga via SQL normally.
```

Comment:
Before does not includes `5.10`. 

This "before" means that "at or during a ​time ​earlier than".

But I think that "Mroonga 5.10 or earlier" is suitable here.
How do you think of that?